app/code/Magento/InventoryInStorePickupOrderExtension/Model/OrderRepository.php

```diff
@@ -0,0 +1,220 @@
+<?php declare(strict_types=1);
```

Comment:
Static tests may fail this way. Mby not travis, but more serious suite we run behind the scenes.

Reply:
Removed.